packages/@aws-cdk/cloudformation-include/lib/cfn-include.ts

```diff
@@ -152,6 +175,38 @@ export class CfnInclude extends core.CfnElement {
     this.parameters[logicalId] = cfnParameter;
   }
 
+  private createOutput(logicalId: string): void {
+    const self = this;
+    const expression = new cfn_parse.CfnParser({
```

Comment:
I don't like the name `expression` here... how about `outputAttributes`, or something like that?